### PR TITLE
Feature/plugin compatibility ie11

### PR DESCRIPTION
**Pull request by swirek-marcin**
Change es6 syntax to be more friendly for unfriendly IE11

### CODE REVIEW

File: assets/js/cg-cookie-consent.js [hunk 1]
```diff
@@ -1,26 +1,24 @@
...
+function cookieExists (name) {
+  return getCookie(name) !== null
 }
 
-function runCookiesPlugin() {
-  var cookiePopupTemplate = window.lodash.template(
-    '<div id="cookiePopup" class="section__cookies" tabindex="-1"> \
+function runCookiesPlugin () {
+
+var cookiePopupTemplate = window.lodash.template(
```
Comment (lechdulian) [inline assets/js/cg-cookie-consent.js]: let's fix indentation here

File: assets/js/cg-cookie-consent.js [hunk 2]
```diff
@@ -52,199 +50,135 @@ function runCookiesPlugin() {
...
-    } finally {
-      if (_didIteratorError) {
-        throw _iteratorError;
-      }
+function runScripts () {
+  const scripts = document.querySelectorAll('script[type="text/plain"]')
+  let customDataLayer = {}
+  window.dataLayer = window.dataLayer || []
+  $(scripts).each(function () {
+    console.log(this)
```
Comment (lechdulian) [inline assets/js/cg-cookie-consent.js]: `console.log` should be removed from production-ready code

File: assets/js/cg-cookie-consent.js (hunk 1 above)
Comment (swirek-marcin) [reply assets/js/cg-cookie-consent.js]: Done ⚡️

File: assets/js/cg-cookie-consent.js (hunk 2 above)
Comment (swirek-marcin) [reply assets/js/cg-cookie-consent.js]: Clean and clear ⚡️

**Pull request merged**